modules/dev-utils/src/main/java/org/apache/ignite/development/utils/IgniteWalConverter.java

```diff
@@ -45,34 +51,95 @@
     public static void main(String[] args) throws Exception {
         if (args.length < 2)
             throw new IllegalArgumentException("\nYou need to provide:\n" +
-                    "\t1. Size of pages, which was selected for file store (1024, 2048, 4096, etc).\n" +
-                    "\t2. Path to dir with wal files.\n" +
-                    "\t3. (Optional) Path to dir with archive wal files.");
+                "\t1. Size of pages, which was selected for file store (1024, 2048, 4096, etc).\n" +
+                "\t2. Path to dir with wal files.\n" +
+                "\t3. (Optional) Path to dir with archive wal files.");
 
+        final int pageSize = Integer.parseInt(args[0]);
+        final String walPath = args[1];
+        final String walArchivePath = (args.length > 2) ? args[2] : null;
+
+        if (F.isEmpty(walPath) && F.isEmpty(walArchivePath))
+            throw new IllegalArgumentException("The paths to the WAL files are not specified.");
+
+        File walDir = null;
+
+        if (!F.isEmpty(walPath)) {
+            walDir = new File(walPath);
+
+            if (!walDir.exists())
+                throw new IllegalArgumentException("Incorrect path to dir with wal files: " + walPath);
+        }
+
+        File walArchiveDir = null;
+
+        if (!F.isEmpty(walArchivePath)) {
+            walArchiveDir = new File(walArchivePath);
+
+            if (!walArchiveDir.exists())
+                throw new IllegalArgumentException("Incorrect path to dir with archive wal files: " + walPath);
+        }
+
+        convert(System.out, pageSize, walDir, walArchiveDir, null, null, true);
+    }
+
+    /**
+     * Write to out WAL log data in human-readable form.
+     *
+     * @param out                           Receiver of result.
+     * @param pageSize                      Size of pages, which was selected for file store (1024, 2048, 4096, etc).
+     * @param walDir                        Path to dir with wal files.
+     * @param walArchiveDir                 Path to dir with archive wal files.
+     * @param binaryMetadataFileStoreDir    Path to binary metadata dir.
+     * @param marshallerMappingFileStoreDir Path to marshaller dir.
+     * @param keepBinary                    Keep binary flag.
+     * @throws IgniteCheckedException If failed.
+     */
+    public static void convert(
+        final PrintStream out,
+        final int pageSize,
+        final File walDir,
+        final File walArchiveDir,
+        final File binaryMetadataFileStoreDir,
+        final File marshallerMappingFileStoreDir,
+        final boolean keepBinary
+
```

Comment:
Please add `skipCrc` to this method's parameters, and also add it to command line parameters, or get from system properties inside of `main` method.

Reply:
Added a stub, the implementation will enter https://ggsystems.atlassian.net/browse/GG-26099